build.cake

```diff
@@ -0,0 +1,198 @@
+#addin "Cake.XdtTransform"
+#addin "Cake.Powershell"
+#addin "Cake.Http"
+#addin "Cake.Json"
+#addin "Newtonsoft.Json"
+
+#load "local:?path=CakeScripts/helper-methods.cake"
+
+
+var target = Argument<string>("Target", "Default");
+var configuration = new Configuration();
+var cakeConsole = new CakeConsole();
+
+var unicornSyncScript = $"./scripts/Unicorn/Sync.ps1";
+
+var configJsonFile = "cake-config.json";
+
+/*===============================================
+================ MAIN TASKS =====================
+===============================================*/
+
+Setup(context =>
+{
+	cakeConsole.ForegroundColor = ConsoleColor.Yellow;
+	
+    var configFile = new FilePath(configJsonFile);
+    configuration = DeserializeJsonFromFile<Configuration>(configFile);
+});
+
+Task("Default")
+.WithCriteria(configuration != null)
+.IsDependentOn("Clean")
+.IsDependentOn("Copy-Sitecore-Lib")
+.IsDependentOn("Publish-Website-Projects")
+.IsDependentOn("Apply-Xml-Transform")
+.IsDependentOn("Modify-Unicorn-Source-Folder")
+.IsDependentOn("Publish-Transforms")
+.IsDependentOn("Sync-Unicorn")
+.IsDependentOn("Deploy-EXM-Campaigns");
+
+Task("Quick-Deploy")
+.WithCriteria(configuration != null)
+.IsDependentOn("Clean")
```

Comment:
Quick-Deploy seems like a duplicate of Publish.
Since .Content has a Quick-Deploy we should probably follow the same naming in this project and remove the "Publish" task?

Reply:
So the remaining tasks will be
1- Default
2- Quick-Deploy
3- Initial

is that correct, @jeanfrancoislarente?

Reply:
Yes